packages/react-api/src/ledger.ts

```diff
@@ -31,8 +31,14 @@ export function clearLedger (): void {
 
 export function getLedger (): Ledger {
   if (!ledger) {
-    ledger = new Ledger(uiSettings.ledgerConn as 'u2f');
+    const def = api && ALLOWED_CHAINS.find(([g]) => g === api.genesisHash.toHex());
```

Comment:
Would allowed_chains now restrict us to kusama?

Reply:
no.. I will update allowed chains to include polkadot too, as 
const ALLOWED_CHAINS: [string, 'kusama' | 'polkadot'][] = [
  [KUSAMA_GENESIS, 'kusama'],
  [POLKADOT_GENESIS, 'polkadot']
];